serverless/functions/serverless-functions-developer-guide.adoc

```diff
@@ -0,0 +1,22 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-functions-developer-guide"]
+= Developer guide
+:context: serverless-functions-developer-guide
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+This guide explains how you can create and manage a function on an {ServerlessProductName} installation by using the `kn` CLI.
+// TODO: add info about developer console at 4.8 when this is supported.
+
+[id="serverless-functions-developer-guide-prerequisites"]
```

Comment:
Thanks, I wasn't aware of that change to the guidelines since it just happened in March; opening issue https://issues.redhat.com/browse/SRVCOM-1296 to update the other sections, but I'll update these headings for now.